app/controllers/application_controller.rb

```diff
@@ -81,7 +81,7 @@ def recaptcha_is_invalid?(model)
   end
 
   def request_event
-    RequestEvent.new(request, response, session, current_jobseeker, current_publisher_oid)
+    RequestEvent.new(request, response, session, current_jobseeker, current_publisher&.oid)
```

Comment:
Can we refactor the `RequestEvent` to take the current publisher the same way it takes the current jobseeker?